rasa/plugin.py

```diff
@@ -143,3 +143,8 @@ def init_anonymization_pipeline(endpoints_file: Optional[Text]) -> None:
 @hookspec(firstresult=True)  # type: ignore[misc]
 def get_anonymization_pipeline() -> Optional[Any]:
     """Hook specification for getting the anonymization pipeline."""
+
+
+@hookspec  # type: ignore[misc]
+def after_server_stop() -> None:
+    """Hook specification for stopping the server."""
```

Comment:
What is this hook for - we should probably make the docstring a bit more explicit?

Reply:
This hook is meant to be used as a trigger for de-initialization of resources in Rasa Pro which need it, like thread shutdown etc.
I updated the comment to be more descriptive.